### ISSUE TITLE
Tuple[int, ...] between parentheses

### ISSUE THREAD

**Issue by maurokenny** [microsoft/pyright#826]
**Describe the bug**
```
from typing import Tuple
from typing import Union
def function_name(parameter1: int,
                  parameter2: int,
                  parameter3: Union[Tuple, int] = ()) -> (
                  Tuple[int, ...]):
...
```
The parentheses at the return type are not allowed, while on mypy it is accepted. It produces the following error: 
`
Tuple expression not allowed in type annotation
  Use Tuple[T1, ..., Tn] instead Pyright
`

**Expected behavior**
The expected behavior is the same without parentheses but with \\, as:
```
def function_name(parameter1: int,
                  parameter2: int,
                  parameter3: Union[Tuple, int] = ()) -> \
                  Tuple[int, ...]:
```
which is accepted without problems

**VS Code extension or command-line**
I am running as an extension on VS Code, version 1.46.1
pyright version 1.1.52

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright & Pylance.

**Comment by erictraut** [comment]
This is now fixed in Pyright 1.1.53, which I just published. It will also be in the next version of Pylance.
